<launch>
<!--   <node pkg="point_cloud_mapping" type="pcd_to_msg_node" name="pcd_to_msg_node" output="screen" respawn="true" -->
<!-- 	args="$(optenv pcd /home/marton/work/datasets/NICO/teapot_at0.pcd) $(optenv period 5)" /> -->

<!--   <node pkg="point_cloud_mapping" type="normal_estimation_node" name="normal_estimation_node" output="screen"> -->
<!--     <param name="search_k_closest" value="30" /> -->
<!--     <param name="downsample" value="0" /> -->
<!--     <remap from="/normal_estimation_node/tilt_laser_cloud" to="/table_object_detector_passive/cloud_annotated"/> -->
<!--     <remap from="/normal_estimation_node/cloud_normals" to="/cloud_pcd_normals"/> -->
<!--   </node> -->

<node pkg="cloud_algos" type="mls_fit_node" name="mls_fit_node" output="screen" respawn="true">
  <remap from="~cloud_pcd" to="/table_object_detector_passive/cloud_annotated" />
</node>


  <node pkg="cloud_algos" type="rotational_estimation_node" name="rotational_estimation_node" output="screen" respawn="true">
    <remap from="~cloud_pcd" to="/mls_fit_node/cloud_mls"/>
    <param name="threshold" type="double" value="0.002"/>
    <param name="probability" type="double" value="0.9999"/>
    <param name="max_iterations" type="int" value="100"/>
  </node>


<!--
 <node pkg="cloud_tools" name="msg_to_pcd" type="msg_to_pcd" respawn="true" output="screen">
   <param name="input_cloud_topic" value="/mls_fit_node/cloud_mls"/>
   <param name="dir" value="$(env HOME)/work/datasets/TMP/"/>
   <param name="nr_saved_pcds" type="int" value="0"/>
 </node>
-->

</launch>
